data/model/test/test_permission.py

```
@@ -93,3 +99,56 @@ def test_get_user_repo_permissions_returns_direct_and_team(initialized_db):
 
     assert "directrepo" in repo_names
     assert "teamrepo" in repo_names
+
+
+def test_batch_team_permission_functions(initialized_db):
```

Ruff (0.14.14)

[warning] Unused function argument: `initialized_db`

(ARG001)